js_modules/dagster-ui/packages/ui-core/src/asset-graph/AssetNode.tsx

```diff
@@ -405,6 +407,18 @@ export const ASSET_NODE_FRAGMENT = gql`
     assetKey {
       ...AssetNodeKey
     }
+    targetingInstigators {
+      ... on Schedule {
+        ...ScheduleSwitchFragment
+      }
+      ... on Sensor {
+        ...SensorSwitchFragment
+      }
+    }
```

Comment:
Is this going to be expensive?

Reply:
This is definitely going to fetch more data than we're fetching right now... but I can't judge whether or not it's prohibitive.

Any tips on how to adjust the fragments based on what facets are enabled?

Reply:
oof... most of the facets are just accessing stuff off of the definition.  We'd probably want to group data fetching chunks together, to avoid exponential explosion of data fetching complexity

Reply:
I'm motivated to find a way to ship this... just don't want it to be throwaway work that has to be rewritten at some point.